### PR TITLE
Implement dao8 hw

### PR DESCRIPTION
**Pull request by IwalkInSilence**

### CODE REVIEW

**Review by AndreiYazadzhi — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -18,28 +18,40 @@
...
         </dependency>
         <dependency>
-            <groupId>javax.servlet</groupId>
-            <artifactId>javax.servlet-api</artifactId>
-            <version>4.0.1</version>
-            <scope>provided</scope>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
```
Comment (dmytro-hryhoruk) [inline pom.xml]: I don't see any tests; Let's remove this

File: src/main/java/core/controllers/car/CreateCarController.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String model = req.getParameter("model");
+        String name = req.getParameter("name");
+        String country = req.getParameter("country");
+        Car car = new Car(model, manufacturerService.create(new Manufacturer(name, country)));
+        carService.create(car);
+        System.out.println(model + " " + name + " " + country);
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/controllers/car/CreateCarController.java]: remove this

File: pom.xml [hunk 3]
```diff
@@ -73,17 +85,7 @@
...
-                    <configuration>
-                        <webXml>src/main/webapp/web.xml</webXml>
-                    </configuration>
-                </plugin>
             </plugins>
         </pluginManagement>
     </build>
-
-
 </project>
```
Comment (dmytro-hryhoruk) [inline pom.xml]: empty line

File: src/main/webapp/WEB-INF/views/driver/all.jsp [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+<body>
+<h1> All driver page </h1>
+<table border="1">
+    <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>License Number</th>
+        <th>Delete</th>
+    </tr>
+    <%--@elvariable id="drivers" type="java.util.List"--%>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/WEB-INF/views/driver/all.jsp]:
```suggestion
```

File: src/main/webapp/WEB-INF/views/manufacture/all.jsp [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+<body>
+<h1> All manufacturers page </h1>
+<table border="1">
+    <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Country</th>
+        <th>Delete</th>
+    </tr>
+    <%--@elvariable id="manufacturers" type="java.util.List"--%>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/WEB-INF/views/manufacture/all.jsp]:
```suggestion
```

File: src/main/webapp/WEB-INF/views/manufacture/all.jsp [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+                <c:out value="${manufacturer.country}"/>
+            </td>
+            <td>
+                <a href="${pageContext.request.contextPath}/mn/delete?id=${manufacturer.id}">Delete</a>
+            </td>
+        </tr>
+    </c:forEach>
+</table>
+</body>
+</html>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/WEB-INF/views/manufacture/all.jsp]: empty line

File: src/main/webapp/web.xml [hunk 7]
```diff
@@ -8,10 +8,99 @@
...
+        <url-pattern>/car/delete</url-pattern>
+    </servlet-mapping>
+
+    <servlet>
+        <servlet-name>addDriverToCar</servlet-name>
+        <servlet-class>core.controllers.driver.AddDriverToCarController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>addDriverToCar</servlet-name>
+        <url-pattern>/car/add</url-pattern>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/web.xml]: https://mate-academy.github.io/jv-program-fulltime/02_jdbc/checklist/08_servlets_practice_checklist.html 
check all url patterns

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

File: src/main/webapp/web.xml [hunk 8]
```diff
@@ -8,10 +8,99 @@
...
+        <url-pattern>/</url-pattern>
+    </servlet-mapping>
+
+    <servlet>
+        <servlet-name>allDrivers</servlet-name>
+        <servlet-class>core.controllers.driver.GetAllDriverController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>allDrivers</servlet-name>
+        <url-pattern>/drivers/all</url-pattern>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/web.xml]:
```suggestion
        <url-pattern>/drivers</url-pattern>
```

File: src/main/webapp/web.xml [hunk 9]
```diff
@@ -8,10 +8,99 @@
...
+        <url-pattern>/manufacturer/delete</url-pattern>
+    </servlet-mapping>
+
+    <servlet>
+        <servlet-name>allCars</servlet-name>
+        <servlet-class>core.controllers.car.GetAllCarsController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>allCars</servlet-name>
+        <url-pattern>/cars/all</url-pattern>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/web.xml]:
```suggestion
        <url-pattern>/cars</url-pattern>
```

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**